substrate/frame/revive/src/evm/api/debug_rpc_types.rs

```diff
@@ -0,0 +1,202 @@
+use crate::{
+	evm::{extract_revert_message, runtime::gas_from_weight, Bytes},
+	ExecReturnValue, Weight,
+};
+use alloc::{string::String, vec::Vec};
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use serde::{Deserialize, Serialize};
+use sp_core::{H160, H256, U256};
+
+/// Tracer configuration used to trace calls.
+#[derive(TypeInfo, Debug, Clone, Encode, Decode, Serialize, Deserialize)]
+pub enum TracerConfig {
+	/// A tracer that captures call traces.
+	CallTracer {
```

Comment:
Will there be more variants or why is it an enum? I also see `LogTracer`

Reply:
potentially, this match the parameters used by go ethereum, they have different flavors of tracers to capture, call traces but also state change, opcode, ...